modules/core/client/views/menu.client.view.html

```diff
@@ -44,15 +44,15 @@
 		<!-- PROJECT CLIENT MENUS (project.client.menus.js) -->
 		<ul class="nav nav-pills nav-stacked">
 			<li ng-repeat="item in menu.projectTopMenu.items | orderBy: 'position'" ng-if="item.shouldRender(menu.authentication.user, menu.project);"
-				ng-class="{ 'active': $state.includes(item.state) }" class="{{item.class}}">
```

Comment:
Did we require this change because we are managing the active state based on the current route, rather than a element/variable in the scope?